backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/RTTIGenerator.kt

```diff
@@ -395,6 +395,30 @@ internal class RTTIGenerator(override val context: Context) : ContextUtils {
     private fun mapRuntimeType(type: LLVMTypeRef): Int =
             runtimeTypeMap[type] ?: throw Error("Unmapped type: ${llvmtype2string(type)}")
 
+    private val debugRuntimeOrNull: LLVMModuleRef? by lazy {
+        context.config.runtimeNativeLibraries.singleOrNull { it.endsWith("debug.bc")}?.let {
+            parseBitcodeFile(it)
```

Comment:
Should be disposed.

Reply:
Alternatively we can avoid loading this module at all.
`debugOperationsSize` could be stored in a runtime-defined struct containing debug operations array (or a pointer to it) too.
Then it would be enough to have a single pointer in `ExtendedTypeInfo` (which can also be considered an optimization).

Reply:
Prefer disposing solution.